framework/src/main/scala/skinny/controller/feature/EnvFeature.scala

```diff
@@ -9,11 +9,12 @@ import org.scalatra.ScalatraBase
 trait EnvFeature extends ScalatraBase {
 
   /**
-   * Env string value from "skinny.env" or "org.scalatra.environment".
+   * Env string value from "skinny.env", "org.scalatra.environment" or default value "development".
```

Comment:
This comment is actually true but that depends on Scalatra's implementation. I think we should not update this.